src/webgpu/api/validation/createBindGroupLayout.spec.ts

```diff
@@ -165,6 +166,27 @@ g.test('multisampled_validation')
     }, !success);
   });
 
+g.test('multisampled_validation,sample_type')
+  .desc('Test that multisampling is not allowed with "float" sample type.')
+  .paramsSubcasesOnly(u => u.combine('sampleType', [undefined, ...kTextureSampleTypes]))
+  .fn(async t => {
+    const { sampleType } = t.params;
+
+    const success = sampleType !== 'float';
```

Comment:
`sampleType` defaults to `"float"`, so I think you need this:
```suggestion
    const success = (sampleType ?? 'float') !== 'float';
```

Reply:
Ah, ok, let me set a fallback value the next time if the spec defines a default value. Thanks!